tree/ntuple/v7/inc/ROOT/RNTupleView.hxx

```diff
@@ -68,6 +69,7 @@ public:
    RNTupleGlobalRange(NTupleSize_t start, NTupleSize_t end) : fStart(start), fEnd(end) {}
    RIterator begin() { return RIterator(fStart); }
    RIterator end() { return RIterator(fEnd); }
+   NTupleSize_t count() { return fEnd - fStart; }
```

Comment:
This is a 'surprising' spelling.  The `begin/end` mimic the STL interface but `count` is not.  Should we use `size` instead? (and/or `ssize`)?